pkg/machines/components/infoRecord.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from "react";
+
+React;
+
+const InfoRecord = ({id, descr, value}) => {
```

Comment:
Almost identical component already exists: `<VmOverviewTabRecord/>` in hostvmslist.jsx